astropy/utils/tests/test_decorators.py

```diff
@@ -57,17 +56,18 @@ def set_private(self):
 
     with pytest.warns(NewDeprecationWarning, match="The bar attribute is "
                       "deprecated and may be removed in a future version.") as w:
-        dummy.bar
+        dummy.bar == 4242
```

Comment:
Do you mean to use `==` instead of `=`?

Reply:
The new commits update the tests properly.